content/en/docs/getting-started/kubernetes-quickstart.md

```diff
@@ -0,0 +1,209 @@
+
+    ```plain 
+    k get pods -n falco  -w
+    ```
+
+You should ultimately see 3 entries similar to below
+
+```plain
+NAME                                   READY   STATUS    RESTARTS   AGE
+falco-falcosidekick-59c5d6cc45-l2qjz   1/1     Running   0          3m7s
+falco-falcosidekick-59c5d6cc45-kpcws   1/1     Running   0          3m7s
+falco-vdsc8                            2/2     Running   0          3m7s
+``` 
+
+## Simulate suspicious activity
+
+One of Falco's rules alerts you if someone runs an interactive shell into a running container. In this section you'll shell into a running Alpine container to trigger a Falco notification
+
+* Start an Alpine container 
+
+    ```plain 
+    kubectl run alpine --image alpine -- sh -c "sleep infinity"
+    ```
+* Wait for the Alpine pod to come online
+
+    ```plain
+    kubectl get pods -w
+    ```
+You should see something similar to this:
+
+```plain
+NAME     READY   STATUS    RESTARTS   AGE
+alpine   1/1     Running   0          16s
+```
+
+* Shell into the runnign container and run the *uptime* command. This will trigger Falco to send an Alert
+
+    ```
+    kubectl exec -it alpine -- sh -c "uptime"
+    ```
+
+## Examine Falco's output
+
+* One of the outputs Falco writes to by default is standard out, so one way to view our notification is to examine the Kubernetes logs for the Falco pod. Since we know our alert has a severity of *Notice* we can grep for that to make it easier to find
+
+```
+kubectl logs -l app.kubernetes.io/name=falco -n falco -c falco | grep Notice
+```
+
+You should see output similar to this:
+
+```
+{"hostname":"falco-vdsc8","output":"22:09:57.827858816: Notice A shell was spawned in a container with an attached
+ terminal (user=root user_loginuid=-1 k8s.ns=default k8s.pod=alpine container=d34690e67a3a shell=sh parent=runc 
+ cmdline=sh -c uptime pid=6461 terminal=34816 container_id=d34690e67a3a image=<NA> exe_flags=EXE_WRITABLE)",
+ "priority":"Notice","rule":"Terminal shell in container","source":"syscall","tags":["T1059","container",
+ "mitre_execution","shell"],"time":"2023-07-26T22:09:57.827858816Z", "output_fields": {"container.
+ id":"d34690e67a3a","container.image.repository":null,"evt.arg.flags":"EXE_WRITABLE","evt.time":1690409397827858816,
+ "k8s.ns.name":"default","k8s.pod.name":"alpine","proc.cmdline":"sh -c uptime","proc.name":"sh","proc.pid":6461,
+ "proc.pname":"runc","proc.tty":34816,"user.loginuid":-1,"user.name":"root"}}
+```
+
+Notice that there is a bunch of interesting information in the output including thigs like the pod name, container id, time of the notification, and much more. 
+
+* Since we used Falcosidekick to send the information to Slack, you can view it there as well. Simply visit our #falco-notifications channel on the [public slack channel](https://join.slack.com/t/sysdig-workshops/shared_invite/zt-1rd6raamc-UldAmUJe5FgnB5NhsAPeMQ) and look for the alert with your name. 
```

Comment:
```suggestion
* Since we used Falcosidekick to send the information to Slack, you can also view it there. Visit our #falco-notifications channel on the [public slack channel](https://join.slack.com/t/sysdig-workshops/shared_invite/zt-1rd6raamc-UldAmUJe5FgnB5NhsAPeMQ) and look for the alert with your name. 
```